analytical_engine/core/grape_instance.cc

```diff
@@ -248,17 +248,25 @@ bl::result<std::string> GrapeInstance::reportGraph(
                   object_manager_.GetObject<IFragmentWrapper>(graph_name));
   auto graph_type = wrapper->graph_def().graph_type();
 
+  VLOG(1) << "Reporting graph, graph name: " << graph_name
```

Comment:
Suggestion, the log could be more informative. 
`Reporting REPORTINGTYPE on graph GRAPHNAME (GRAPHTYPE).`

Reply:
remove the LOG, since reporting graph operation is very frequent